### ISSUE TITLE
Unable to run tailwire:install

### ISSUE THREAD

**Issue by marsderp** [redbastie/tailwire#2]
On a fresh Laravel 8 install, running the install command `php artisan tailwire:install` throws this error:

Is there something required that I am missing? I think it's not creating the necessary folders, like `app/Components`, `app/Components/Layout`, `public/images` and `public/json`. Only after I manually create these folders does the install work.

```
λ php artisan tailwire:install

   ErrorException

  file_put_contents(app\Components\Index.php): failed to open stream: No such file or directory

  at C:\laragon\www\tailwire\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php:185
    181▕      * @return int|bool
    182▕      */
    183▕     public function put($path, $contents, $lock = false)
    184▕     {
  ➜ 185▕         return file_put_contents($path, $contents, $lock ? LOCK_EX : 0);
    186▕     }
    187▕
    188▕     /**
    189▕      * Write the contents of a file, replacing it atomically if it already exists.

  1   C:\laragon\www\tailwire\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php:185
      file_put_contents("app\Components\Index.php", "<?php

namespace App\Components;

use Redbastie\Tailwire\Component;
use Redbastie\Tailwire\View;

class Index extends Component
{ ...
```

**Comment by redbastie** [comment]
What operating system?

**Comment by redbastie** [comment]
Nvm I see you're on windows. I'll see if I can figure it out.

**Comment by marsderp** [comment]
Don't have any special permissions on Windows for Laragon's www folder.

**Comment by redbastie** [comment]
I figured it out. It has to do with the directory separator. I'm working on a fix and will push soon.

**Comment by redbastie** [comment]
This is now fixed in 2.0.4. I just released it. Hope this helps.
